content/docs/apps/node.md

```diff
@@ -21,6 +21,14 @@ Deploying a basic node.js application to Cloud Foundry is relatively uncomplicat
     ```
 
 - Avoid using the `command:` manifest key to start your application, as this functionality [may be removed](https://github.com/cloudfoundry/nodejs-buildpack/pull/11#issuecomment-67666273). The aforementioned npm start script is preferred.
+
+- Vendor your dependencies.  Cloud Foundry [documentation](https://docs.cloudfoundry.org/buildpacks/node/index.html#vendoring) specifies that you should do a `npm install` to install vendored dependencies and pushing your entire application with `cf push`. This is especially
+important for large applications as Cloud Foundry limits the size of an app upload to `1GB` -
```

Comment:
It's important to differentiate here between AWS E/W and GovCloud. Also, the limit is a default of `1GB` unless more space is given using the `disk_space: <Value>` property in the manifest. Maybe something like GovCloud application containers have a default disk limit of 1GB. But I'd verify this in `#cloud-gov-atlas` to be sure.